scripts/build_native_lib_in_clean_repo.sh

```diff
@@ -31,7 +31,7 @@ echoerr "Moving to $PROJECT_TMP_DIR"
 cd "$PROJECT_TMP_DIR"
 
 echoerr "cloning broadinstitute/${PROJECT}"
-GIT_LFS_SKIP_SMUDGE=1 git clone git@github.com:broadinstitute/${PROJECT}.git 1>2
+GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/broadinstitute/${PROJECT}.git 1>2
```

Comment:
`https://` clones always fail for me on the broad servers, there's probably something wrong with whatever git dotkit I happen to be using though.  You're the one who's probably running this in the future, so if it works for you then I have no objection to it changing.